src/ynEIGEN/ynEigenDepositAdapter.sol

```diff
@@ -84,6 +87,10 @@ contract ynEigenDepositAdapter is IynEigenDepositAdapterEvents, Initializable, A
         oETH = IERC20(woETH.asset());
     }
 
+    function initializeV2(address _wrapper) external reinitializer(2) notZeroAddress(_wrapper) {
```

Comment:
we are assuming this runs as part of the upgradeToAndCall

Reply:
right